shared/health/src/main/java/io/pravega/shared/health/ContributorRegistry.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.health;
+
+import io.pravega.shared.health.impl.HealthComponent;
+
+import java.util.Collection;
+
+/**
+ * Holds the set of {@link HealthContributor} objects that will be tracked to determine
+ * the overall state of some {@link HealthService}.
+ */
+public interface ContributorRegistry extends Registry<HealthContributor> {
+
+    static final String DEFAULT_CONTRIBUTOR_NAME = "health-service";
+
+    HealthContributor get();
```

Comment:
Javadoc. Also, what is the point of this method?

Reply:
Supplies the root level `HealthContributor`. Will change to a more appropriate name.